go/cmd/zk/zkcmd.go

```diff
@@ -133,35 +130,41 @@ func init() {
 	}
 }
 
-var (
-	server = flag.String("server", "", "server(s) to connect to")
-)
-
 func main() {
 	defer exit.Recover()
 	defer logutil.Flush()
+	server = pflag.String("server", "", "server(s) to connect to")
+
+	// handling case of --help, -help & -h
+	args := os.Args[1:]
+	var help bool
+	if args[0] == "-h" {
+		pflag.BoolVarP(&help, "help", "h", false, "display usage and exit")
+	} else if args[0] == "--help" || args[0] == "-help" {
+		pflag.BoolVar(&help, "help", false, "display usage and exit")
+	}
 
-	fs := pflag.NewFlagSet("zkcmd", pflag.ExitOnError)
-	log.RegisterFlags(fs)
-	logutil.RegisterFlags(fs)
```

Comment:
you've lost the log/logutil flags, i think

Reply:
zkcmd is non-vitess binary. So I don't think it needs vitess flags... Therefore I kept it simple.. I have added it in description as well.

Reply:
it never calls into `log` functions? i find that surprising

Reply:
How do I add log.RegisterFlags here given i am not using flagset and is using pure pflag fucntions..

Reply:
`log.RegisterFlags(pflag.CommandLine)`